app/assets/javascript/app/actions/api.js

```diff
@@ -25,6 +25,7 @@ async function postJSON(url, body) {
     method: 'POST',
     body,
   });
+  console.log(response)
```

Comment:
Do we want to keep `console.log` statements in prod?

Reply:
Nope